blueocean-dashboard/src/main/js/creation/github/GithubFlowManager.js

```diff
@@ -52,40 +50,15 @@ export default class GithubFlowManager extends FlowManager {
         if (!this.repositories) {
             return [];
         }
-
-        if (!this.existingOrgFolder || !this.existingOrgFolder.pipelineFolderNames) {
-            return this.repositories;
-        }
-
-        // return repositories that are not already created as pipelines
-        return this.repositories.filter(repo => (
-            this.existingOrgFolder.pipelineFolderNames.indexOf(repo.name) === -1
-        ));
+        return this.repositories;
```

Comment:
Perhaps we should just refactor all references to `selectableRepositories` to `repositories` instead?